src/diagnostics.ts

```diff
@@ -101,23 +112,41 @@ export function loadDiagnostic(context: d.PluginCtx, sassError: SassException, f
   return diagnostic;
 }
 
-function formatCode(input: number) {
+/**
+ * Helper function for converting a number error code to a string
+ * @param input the numeric error code to convert
+ * @returns the stringified error code
+ */
+function formatCode(input: number): string {
```

Comment:
while this shouldn't happen per the type system, I left the nullish check in here since this was largely a documentation effort

Reply:
I think since we don't have `"strict": true` in this project `input` could be called with `undefined` and TypeScript wouldn't let us know, so the check might be wise. Although to be fair, just inlining `String(num ?? "")` would accomplish the same thing as this function anyway

Reply:
Ah, yeah...true